deb/build

```diff
@@ -66,41 +105,41 @@ fpm -s dir -t deb -n "$pkgname" -v "$pkgversion" \
     --depends libgcc1 \
     --deb-changelog "$changelog" \
     --deb-no-default-config-files \
-    $so_libname.$libversion=/usr/lib/ \
-    $so_libname.$lib_major=/usr/lib/
+    $so_libname.$libversion=/usr/lib/$dist_pkgname.so.$libversion \
+    $libdir/$dist_so_libname.$lib_major=/usr/lib/
 
-pkgname=libdrizzle-redux-dbg
-genchangelog "$pkgname" "$pkgversion" "$pkgmaint" > "$changelog"
-build_id=`readelf -n install/usr/lib/libdrizzle-redux.so.$libversion | \
+# libdrizzle-redux debug package
+genchangelog "$dist_pkgname-dbg" "$pkgversion" "$pkgmaint" > "$changelog"
+build_id=`readelf -n $so_libname.$libversion | \
     sed -n 's/^.*Build ID: \([a-f0-9]\{40\}\).*$/\1/p'`
 debug_file=`printf $build_id | cut -b1-2`/`printf $build_id | cut -b3-`.debug
 objcopy --only-keep-debug $so_libname.$libversion.full $so_libname.$libversion.debug
-fpm -s dir -t deb -n "$pkgname" -v "$pkgversion" \
+fpm -s dir -t deb -n "$dist_pkgname-dbg" -v "$pkgversion" \
     --architecture all \
     --maintainer "$pkgmaint" \
     --description "Simplified API to MySQL databases" \
     --url 'https://github.com/sociomantic-tsunami/libdrizzle-redux' \
     --vendor 'Sociomantic Labs GmbH' \
     --license 'Simplified BSD License' \
     --category debug \
-    --depends libdrizzle-redux \
+    --depends $dist_pkgname \
     --deb-changelog "$changelog" \
     --deb-no-default-config-files \
     $so_libname.$libversion.debug=/usr/lib/debug/.build-id/$debug_file
 
-pkgname=libdrizzle-redux-dev
-genchangelog "$pkgname" "$pkgversion" "$pkgmaint" > "$changelog"
-fpm -s dir -t deb -n "$pkgname" -v "$pkgversion" \
+# libdrizzle-redux development package
+genchangelog "$dist_pkgname-dev" "$pkgversion" "$pkgmaint" > "$changelog"
+fpm -s dir -t deb -n "$dist_pkgname-dev" -v "$pkgversion" \
     --architecture all \
     --maintainer "$pkgmaint" \
     --description "Simplified API to MySQL databases" \
     --url 'https://github.com/sociomantic-tsunami/libdrizzle-redux' \
     --vendor 'Sociomantic Labs GmbH' \
     --license 'Simplified BSD License' \
     --category libdevel \
-    --depends libdrizzle-redux \
+    --depends $dist_pkgname \
     --deb-changelog "$changelog" \
     --deb-no-default-config-files \
-    install/usr/include/=/usr/ \
-    $so_libname=/usr/lib/ \
-    install/usr/lib/libdrizzle-redux.a=/usr/lib/
+    install/usr/include/include/=/usr/include/$pkgname/$dist_pkgname/ \
```

Comment:
Yes, with the new project layout the make install create a two `include` subdirs.